<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaElch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/MediaElch.qrc">
    <normaloff>:/img/MediaElch.png</normaloff>:/img/MediaElch.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="SlidingStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="MovieSearchWidget" name="movieSearchWidget"/>
     <widget class="QWidget" name="importPage2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1,0">
         <item>
          <widget class="QLabel" name="loading">
           <property name="text">
            <string>Loading</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Badge" name="badgeSuccess">
           <property name="text">
            <string>Success</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelLoading">
           <property name="text">
            <string>Loading movie...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="RenamerPlaceholders" name="placeholders" native="true"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelDirectoryNaming">
           <property name="text">
            <string>Directory Naming</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="directoryNaming"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>File Naming</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="fileNaming"/>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="chkSeasonDirectories">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelUseSeasonDirectories">
           <property name="text">
            <string>Use Season Directories</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelSeasonNaming">
           <property name="text">
            <string>Season Directory Naming</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="seasonNaming"/>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="chkKeepSourceFiles">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Keep source files after import</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="ConcertSearchWidget" name="concertSearchWidget"/>
     <widget class="TvShowSearchWidget" name="tvShowSearchWidget"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnReject">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccept">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>ui/small_widgets/SlidingStackedWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Badge</class>
   <extends>QLabel</extends>
   <header>ui/small_widgets/Badge.h</header>
  </customwidget>
  <customwidget>
   <class>MovieSearchWidget</class>
   <extends>QWidget</extends>
   <header>ui/movies/MovieSearchWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConcertSearchWidget</class>
   <extends>QWidget</extends>
   <header>ui/concerts/ConcertSearchWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TvShowSearchWidget</class>
   <extends>QWidget</extends>
   <header>ui/tv_show/TvShowSearchWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenamerPlaceholders</class>
   <extends>QWidget</extends>
   <header>renamer/RenamerPlaceholders.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnReject</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAccept</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
